lib/node_modules/@stdlib/assert/is-alphanumeric/README.md

```diff
@@ -0,0 +1,136 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# isAlphaNumeric
+
+> Test whether a string contains only numeric digits.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var isAlphaNumeric = require( '@stdlib/assert/is-alphanumeric' );
+```
+
+#### isAlphaNumeric( value )
+
+Tests whether a string contains only alphanumeric characters.
+
+```javascript
+var bool = isAlphaNumeric( 'abc0123456789' );
+// returns true
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   For non-string values, the function returns `false`.
+-   For the purposes of this function, **alphanumeric** is defined to be a collection of letters and numbers.
```

Comment:
Can we rephrase to something like `**alphanumeric** is defined as a combination of the letters of the English alphabet and the numbers 0 to 9`?

Reply:
Resolved!